game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/StrengthValue.java

```diff
@@ -0,0 +1,26 @@
+package games.strategy.triplea.delegate.power.calculator;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Value;
+
+/**
+ * Holds the value of a dice strength
```

Comment:
nit, a die does not have strength. 
Rephrasing suggestion: ` Holds the value of units dice roll strength`

Reply:
"roll" has another meaning already that means "how many rolls a unit is allowed".  I wouldn't want to use "roll" here since it is confusing terminology.

Reply:
Good point. We should fix "dice strength", a die does not have strength.

Perhaps a "units dice strength" (?)

Or call it the "ToHitStrength". Or, since we've defined power and strength, we perhaps could just call it a units "strength" and it should then be clear as we have defined strength.